addons/im_livechat/static/src/models/livechat_button_view.js

```diff
@@ -763,6 +778,10 @@ registerModel({
         isChatbotRedirecting: attr({
             default: false,
         }),
+        isLastMessageFromCustomer: attr({
```

Comment:
strange to have this field on button view model? It only uses global concerns it looks like, and it is only used in the context of chatbot?

Reply:
Indeed, makes sense to define in `PublicLivechatGlobal`.
Even though it's only used in chatbot right now, it's not a chatbot specific concept, as you can see by its compute method.